app/lego.py

```diff
@@ -141,17 +142,25 @@ def monitor():
         threading.Thread(target=monitor, name="monitor").daemon = True
         threading.Thread(target=monitor, name="monitor").start() 
 
-    def startMp3(self, filename):
+    def startMp3(self, filename, is_playlist=False):
         global mp3_duration
         # load an mp3 file
-        mp3file = os.path.dirname(os.path.abspath(__file__)) + '/../music/' + filename
-        logger.info('Playing %s.' % filename)
-        self.p.open(mp3file)
-        self.p.play()
+        if not is_playlist:
+            mp3file = os.path.dirname(os.path.abspath(__file__)) + '/../music/' + filename
```

Comment:
Change to:
`mp3file = mp3_dir + filename`

Reply:
ok, done...I also put the mp3_dir as input variable in def startMp3

`def startMp3(self, filename, mp3_dir, is_playlist=False):`